mea/src/once/once_cell.rs

```diff
@@ -46,7 +46,7 @@ use crate::internal;
 pub struct OnceCell<T> {
     value_set: AtomicBool,
     value: UnsafeCell<MaybeUninit<T>>,
-    semaphore: internal::Semaphore,
+    semaphore: Semaphore,
```

Comment:
Previously I was considering using the inernal Semaphore's `notify_all` method, but it seems a simply P/V pattern is enough here. Switch to the higher level encapsulation.